examples/multimodal_v1/components/direct_processor.py

```
@@ -0,0 +1,215 @@
+        async for response in await self._stream_response(
+            request, output, request_id, conversation
+        ):
+            yield response
+
+    # This method is used to process the responses from the engine generator.
+    async def _generate_responses(
+        self,
+        response_generator: AsyncIterator[RequestOutput],
+        request_type: RequestType,
+    ) -> AsyncIterator[Union[RequestOutput, Tuple[int, RequestOutput]]]:
+        prompt_idx = 0
```

Ruff (0.12.2)

Local variable `prompt_idx` is assigned to but never used

Remove assignment to unused variable `prompt_idx`

(F841)